orangepiplus-next.report

```diff
@@ -0,0 +1,9 @@
+BOOT=yes
+VERSION=5.60
+KERNEL=4.14.70-sunxi
+NETWORK=NT
```

Comment:
can you please test network as well? It's something most people use.

Reply:
Yes I will on Monday, when I get back to my workshop. I am at the mercy of mobiles internet tethering atm. and I don't have even a single rj45 at hand. BTW. Thank you for your quick and very helpful answer in the forums. I would be more than happy to do some more testing if required. I have two Orange Pi+'s to play with. Another one is running my 3D printer and it's frontend/backend (Octoprint+klipper), but the other one is solely for development for now.

Reply:
No hurry. ;) I would use the board which isn't in productive.. ;) Or just insert an SD-Card in case your productive OS is on eMMC, bootorder will always boot SD-first, so as long as there's no nand-sata-install call.. it should only mess your SD card. ;)